hoodie-client/src/main/java/com/uber/hoodie/config/HoodieIndexConfig.java

```diff
@@ -50,6 +50,9 @@
   public static final String BLOOM_INDEX_INPUT_STORAGE_LEVEL =
       "hoodie.bloom.index.input.storage" + ".level";
   public static final String DEFAULT_BLOOM_INDEX_INPUT_STORAGE_LEVEL = "MEMORY_AND_DISK_SER";
+  public static final String BLOOM_INDEX_ENABLE_DYNAMIC_PROP =
```

Comment:
rename to "hoodie.bloom.index.dynamic.bloomfilter" or "hoodie.bloom.index.auto.tune.bloomfilter" ? to make it clearer it's about the bloom filter and not the index checking.. 

Also this should probably belong in storage config? since its about how we write the parquet file right

Reply:
I suggest we leave it in the IndexConfig since logically that's where people would look for all Index/BloomFilter related settings ?

Reply:
depends on how you look at it.. At the code level, its weird to suddenly access an index config in storage.. we can leave it here for now. but let's rename?